### PR TITLE
CU-fyrkec_mobile-registration-page

### PR DESCRIPTION
**Pull request by bmitch242**
Designed mobile registration page

### CODE REVIEW

**Comment by bmitch242** [comment]
Task linked: [CU-fyrkec mobile registration page](https://app.clickup.com/t/fyrkec)

**Comment by carteralbrecht** [comment]
Task linked: [CU-fyrkec mobile registration page](https://app.clickup.com/t/fyrkec)

**Review by cjack0416 — APPROVED** [review]
Looks good!

File: mobile/client/src/screens/Registration.js [hunk 1]
```diff
@@ -0,0 +1,166 @@
+import React from 'react';
+import {StyleSheet, Text, TextInput, TouchableOpacity, View} from 'react-native';
+
+class Reg extends React.Component {
+    constructor() {
+    super();
+    this.state = {
+      username: "",
```
Comment (carteralbrecht) [inline mobile/client/src/screens/Registration.js:8]: Should we change to "nickname"? That is what the web app uses as of rn

Comment (carteralbrecht) [reply mobile/client/src/screens/Registration.js:8]: Could change the web app to username also

Comment (cjack0416) [reply mobile/client/src/screens/Registration.js:8]: Oh yeah, if we want to keep things consistent then that makes sense.

**Pull request merged**